src/ssvc/decision_points/base.py

```diff
@@ -187,6 +187,14 @@ class DecisionPoint(
     def __str__(self):
         return FIELD_DELIMITER.join([self.namespace, self.key, self.version])
 
+    @property
+    def id(self):
+        """
+        Return an identity string for the DecisionPoint.
+        """
+
+        return FIELD_DELIMITER.join([self.namespace, self.key, self.version])
```

Comment:
Just a word of caution here: The `/` in the namespace has probably to be quoted if the you want to use as property name inside a JSON path expression.
But that doesn't stop us, right?